src/components/display/fewlines/Form/Form.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+type FormProps = {
+  onSubmit: (event?: React.FormEvent<HTMLFormElement>) => Promise<void>;
+  children: React.ReactNode | React.ReactNode[];
+};
+
+export const Form: React.FC<FormProps> = ({ onSubmit, children }) => {
+  const [isNotSubmitted, setIsNotSubmitted] = React.useState(true);
```

Comment:
Why don't initialise the form with `isSubmitted = false` instead of `isNotSubmitted = true`, it seems more effective

Reply:
Aaahh glad to see you here mate!
Well, it's quite the same, but I just preferred not to do
```
if (!isSubmitted) {
  await onSubmit();
}
```

Reply:
I'm in the process of writing the tests for the Form, and trying some stuff with Enzyme.